### PR TITLE
O3-1216 dashboard list accordion

### PR DESCRIPTION
**Pull request by pirupius**
https://user-images.githubusercontent.com/15266028/184661765-48c1206f-f833-4d30-847f-c42b16594f19.mov

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,7 +57,7 @@
     "rxjs": "6.x"
   },
   "devDependencies": {
-    "@openmrs/esm-framework": "next",
+    "@openmrs/esm-framework": "latest",
```
Comment (pirupius) [inline package.json:60]: changed to `latest` because `next` has the `4.0 (React 18, Router 6 and Carbon 11)` changes merged in.

**Review by cioan — APPROVED** [review]
LGTM! Thanks @pirupius !

File: src/components/prescription-details.component.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  const { medications, isError, isLoading } = useOrderDetails(encounterUuid);
+
+  return (
+    <div className={styles.prescriptionContainer}>
+      <Tile className={styles.allergiesTile}>
+        <div className={styles.allergesContent}>
+          <div>
+            <WarningFilled24 className={styles.allergiesIcon} />
+            <p>
+              <b>3 allergies</b> Penicillin, Naproxen sodium, Ibuprofen
```
Comment (mseaton) [inline src/components/prescription-details.component.tsx:22]: We need to be careful about inserting dummy data into these components, especially data like this.

**Review by mseaton — APPROVED** [review]
Generally looks great to me, nice work!  Only one comment about dummy data, might want to replace that with some text that clearly indicates this is not real data.

File: src/components/prescription-details.component.tsx (hunk 2 above)
Comment (pirupius) [reply src/components/prescription-details.component.tsx:22]: Understood. Will rectify this in subsequent PRs

**Comment by jonathandick** [comment]
@pirupius could you possible update the CI so that this deploys to dev3?

**Comment by pirupius** [comment]
Sure thing @jonathandick

**Pull request merged**